src/main/java/uk/nhs/hee/trainee/details/api/TraineeProfileResource.java

```diff
@@ -144,4 +144,21 @@ public ResponseEntity<String> getVerifiedTraineeIds(@NotNull @RequestParam Strin
       return ResponseEntity.ok(traineeIds.get(0));
     }
   }
+
+  /**
+   * Get the trainee email for a trainee TIS ID.
+   *
+   * @param id The TIS ID to search by.
+   * @return The matching trainee email.
+   */
+  @GetMapping("/trainee-email")
```

Comment:
We generally use a simple path param instead of a request param for ids

Reply:
D'oh. Yes, of course!